backend/src/v5/models/users.js

```diff
@@ -207,4 +209,35 @@ User.getAvatar = async (username) => {
 
 User.uploadAvatar = (username, avatarBuffer) => updateUser(username, { $set: { 'customData.avatar': { data: avatarBuffer } } });
 
+User.resetPasswordToken = async (username) => {
+	let email; let resetPasswordToken; let
+		firstName;
+
+	try {
+		const { customData } = await User.getUserByQuery({ user: username }, { user: 1, 'customData.email': 1, 'customData.firstName': 1 });
```

Comment:
Don't need the next 2 lines

```suggestion
		const { customData: { email, firstName } } = await User.getUserByQuery({ user: username }, { user: 1, 'customData.email': 1, 'customData.firstName': 1 });
```